docs/releases/1.10.txt

```diff
@@ -26,6 +26,17 @@ What's new in Django 1.10
 
 ...
 
+Bug fixes
```

Comment:
Ok, I into `Backwards incompatible changes`  because I bet there is someone who passed an open file to a response and tried to access it afterwards. I extended the description raise awareness for this issue.